src/cli/commands/wallet/import.rs

```diff
@@ -45,6 +55,12 @@ impl CommandLineHandler for WalletImport {
 pub(crate) struct WalletImportArgs {
     #[arg(long, short, help = "The JSON RPC server url for ipc agent")]
     pub ipc_agent_url: Option<String>,
+    #[arg(
+        long,
+        short,
+        help = "The network the key belongs to, 0 for fvm, 1 for evm"
+    )]
+    pub network_type: u8,
```

Comment:
Would it be possible to make it an enum or even two different boolean flags `--fvm` and `--evm`?